doc/ref_impl/cntt-ri2/chapters/chapter04.rst

```diff
@@ -175,6 +175,23 @@ follows:
      "pods": "110"
    }
 
+Installation on Provider Infrastructure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The main steps are the same as :ref:`chapters/chapter04:Installation on Bare Metal Infratructure`.
+
+Make sure the infrastructure preprovisioned follows ``docs/release/installation/deployment-guide.rst``.
+
+When modifying the environmental variables defined in ``deploy.env``,
+set deployment type ``DEPLOYMENT:-k8s``.
+If you are not the root user account, you neet set ``USERNAME`` to consistent the username of VM and yours,
```

Comment:
Please desrcibe how to set these params and what should be the values of them.

Reply:
done